src/contexts/shoppingListsContext.tsx

```diff
@@ -1,30 +1,40 @@
 import { createContext, useEffect, useState, useCallback } from 'react'
 import { signOutWithGoogle } from '../firebase'
-import { type ResponseShoppingList as ShoppingList } from '../types/apiData'
+import { CallbackFunction } from '../types/functions'
+import {
+  type RequestShoppingList,
+  type ResponseShoppingList,
+} from '../types/apiData'
 import { type ProviderProps } from '../types/contexts'
-import { getShoppingLists } from '../utils/api/simApi'
+import { ApiError } from '../types/errors'
+import { LOADING, DONE, ERROR, type LoadingState } from '../utils/loadingStates'
+import { postShoppingLists, getShoppingLists } from '../utils/api/simApi'
+import { useQueryString } from '../hooks/useQueryString'
 import {
   useGoogleLogin,
   usePageContext,
   useGamesContext,
 } from '../hooks/contexts'
-import { useQueryString } from '../hooks/useQueryString'
-import { ApiError } from '../types/errors'
-import { LOADING, DONE, ERROR, type LoadingState } from '../utils/loadingStates'
 
 const NOT_FOUND_MESSAGE =
-  "You have requested shopping lists for a game that doesn't exist, or doesn't belong to you. Please select another game and try again."
+  "The game you've selected doesn't exist, or doesn't belong to you. Please select another game and try again."
```

Comment:
I've changed this because, if the user has tried to create a shopping list, a nontechnical user would not consider that "requesting" a shopping list.